docs/infrastructure/microk8s/install.md

```diff
@@ -0,0 +1,274 @@
+# Installing Codacy on microk8s
+
+## Pre-requisites
+
+* a machine running Ubuntu 18.04 LTS version.
+  * Further versions might work but we recommend using an LTS release.
+* a remote session (ssh) onto your machine of choice.
+
+## Cookbook
+
+Assuming that you are starting from a blank slate, the first step is to install microk8s. Otherwise, please jump on to the [Configuring microk8s](###Configuring-microk8s) section.
+
+### Installing microk8s
```

Comment:
I would recommend to link to the upstream documentation for installing microk8s instead of rolling out our own.
Ideally we should solely maintain the diff that is needed for Codacy.(e.g. the installation of `nfs-common`).
https://microk8s.io/docs/
WDYT?

Reply:
Good point!